test/server/core/gameSteps/actionWindow/Initiative.spec.ts

```diff
@@ -22,16 +22,12 @@ describe('Claiming initiative', function() {
                 context.player1.claimInitiative();
                 expect(context.player2).toHaveExactPromptButtons(['Pass']);
                 expect(context.ardentSympathizer.getPower()).toBe(5);
-                // TODO atm we do not list the set of cards with available actions in the action phase to bypass the error
-                // TODO Expected Player prompt to change.
-                context.player2.clickCardNonChecking(context.scoutBikePursuer);
+                context.player2.clickCard(context.scoutBikePursuer);
```

Comment:
I know this happened at least a few other times with other folks, do you mind briefly investigating other uses of `clickCardNonChecking` to see if they could be switched out as well?

Reply:
I'd love to.